cluster/hooks.go

```diff
@@ -56,38 +56,36 @@ import (
 )
 
 //RunPostHooks calls posthook functions with created cluster
-func RunPostHooks(postHooks []PostFunctioner, cluster CommonCluster) (err error) {
+func RunPostHooks(postHooks []PostFunctioner, cluster CommonCluster) error {
 
 	log := log.WithFields(logrus.Fields{"cluster": cluster.GetName(), "org": cluster.GetOrganizationId()})
 
 	for _, postHook := range postHooks {
 		if postHook != nil {
+			log := log.WithField("postHook", postHook)
 			log.Infof("Start posthook function[%s]", postHook)
-			err = postHook.Do(cluster)
-			if err != nil {
+			if err := postHook.Do(cluster); err != nil {
 				log.Errorf("Error during posthook function[%s]: %s", postHook, err.Error())
```

Comment:
If the error is returned and logged in an upper level in the code, maybe we can remove this log?